lib/Common/CommonPal.h

```diff
@@ -119,6 +119,28 @@ inline void DebugBreak()
 // activscp.h
 #define SCRIPT_E_RECORDED                _HRESULT_TYPEDEF_(0x86664004L)
 
+
+typedef
+enum tagBREAKPOINT_STATE
```

Comment:
Confused- why is this needed? Are we planning on supporting IDebugDocument on Linux? Shouldn't the code that depends on this be just ifdefed out?